packages/garbo/src/fights.ts

```diff
@@ -7,6 +7,7 @@ import {
   canEquip,
   cliExecute,
   closetAmount,
+  count,
```

Comment:
I have no idea what the kolmafia `count` function does but if I had to guess it's not what you want it to do here specifically